app/routes/capacity-route.js

```diff
@@ -2,11 +2,25 @@ const getCapacityView = require('../services/get-capacity-view')
 const dateRangeHelper = require('../services/helpers/date-range-helper')
 const getSubNav = require('../services/get-sub-nav')
 const organisationUnit = require('../constants/organisation-unit')
+const ValidationError = require('../services/errors/validation-error')
 const getOrganisationUnit = require('../services/helpers/org-unit-finder')
 
 module.exports = function (router) {
   router.get(`/:organisationLevel/:id/caseload-capacity`, function (req, res, next) {
-    var capacityDateRange = dateRangeHelper.createCapacityDateRange(req.query)
+    var capacityDateRange
+    var errors
+
+    try {
+      capacityDateRange = dateRangeHelper.createCapacityDateRange(req.query)
+    } catch (error) {
+      if (error instanceof ValidationError) {
+        errors = error.validationErrors
+        capacityDateRange = dateRangeHelper.createCapacityDateRange({})
```

Comment:
Where is this capacityDateRange being used. I see its declaration and assignment don't see where its been used?

Reply:
It's line 30. It was already there before this PR, hence it's not in the diff.

Reply:
Ok